search-website/README.md

```diff
@@ -19,6 +19,8 @@ This Python sample shows you how to add document search to a web app.
 
 For this sample, you will use the Azure.Search.Documents library in the Azure SDK for Python to create, load, and query a search index containing the goodbooks-10k dataset, publicly available at [https://github.com/zygmuntz/goodbooks-10k](https://github.com/zygmuntz/goodbooks-10k). The search index runs on an [Azure Cognitive Search](https://docs.microsoft.com/azure/search/search-what-is-azure-search) service that you create. You can use the free tier for this sample.
 
+Optionally, this sample includes a devcontainer.json file so that you can run it as a containerized app. You'll need the [Docker extension](https://code.visualstudio.com/docs/containers/overview) to do this. If you don't want Docker, run your code in a virtual environment instead.
```

Comment:
Docker is really about running the code locally, as a developer, with the assurance that the environment is correctly configured and your local dev box doesn't need anything beyond docker. It isn't about running a "containerized app". 

The devcontainer.json file solves the issue of "it ran on my box, don't know why it doesn't run on yours".